common/lib/container-definitions/src/runtime.ts

```diff
@@ -121,7 +121,9 @@ export interface IContainerContext extends IDisposable {
     readonly quorum: IQuorum;
     readonly audience: IAudience | undefined;
     readonly loader: ILoader;
+    /** @deprecated - use taggedLogger instead */
     readonly logger: ITelemetryBaseLogger;
+    readonly taggedLogger?: ITelemetryBaseLogger;
```

Comment:
It would be great to add at least some comments that explain the expectations, the differences, etc.

Reply:
Still do not see comment on expectations for taggedLogger :)